public_html/app/Tools/Request.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace VFramework\Tools;
+
+class Request
+{
+    private $method = '';
```

Comment:
I think better to have variable with default value for example
`private $method = 'GET'`

Reply:
and maybe change it to const